src/h5web/providers/my-utils.ts

```diff
@@ -176,7 +178,7 @@ export function makeMyNxDataGroup<
         ...(groupOpts.attributes || []),
         makeStrAttr('NX_class', 'NXdata'),
         makeStrAttr('signal', signal.name),
-        makeStrAttr('axes', axesAttr),
+        ...(axesAttr ? [makeNxAxesAttr(axesAttr)] : []),
```

Comment:
`makeStrAttr` was accepting `any` value, so we were creating an attribute of type/shape string/scalar but with an array as value... Since we never look at the type/shape of attributes, this had no impact on the app.

I was also creating the attribute regardless of whether `axesAttr` was provided... 🤦

Reply:
:fishing_pole_and_fish: